virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -6,67 +6,52 @@ include::_attributes/common-attributes.adoc[]
 
 [WARNING]
 ====
-Do not create virtual machines in `openshift-*` namespaces.
+Do not create virtual machines in the default `openshift-*` namespaces.
 Instead, create a new namespace or use an existing namespace without the `openshift` prefix.
 ====
 
-When you create virtual machines from the web console, select a virtual machine template that is configured with a boot source. Virtual machine templates with a boot source are labeled as *Available boot source* or they display a customized label text. Using templates with an available boot source expedites the process of creating virtual machines.
+[id="creating-vm-web-_{context}"]
+== Creating a virtual machine by using the web console
 
-Templates without a boot source are labeled as *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
+You create a virtual machine (VM) from a template, YAML file, or instance type by using the {product-title} web console.
 
 [IMPORTANT]
 ====
 Due to differences in storage behavior, some virtual machine templates are incompatible with {sno}. To ensure compatibility, do not set the `evictionStrategy` field for any templates or virtual machines that use data volumes or storage profiles.
 ====
 
-include::modules/odc-creating-vms.adoc[leveloffset=+1]
-
-include::modules/virt-creating-vm-quick-start-web.adoc[leveloffset=+1]
+Virtual machine templates with a boot source are labeled *Available boot source* or they display a customized label text. Using templates with an available boot source expedites the process of creating virtual machines.
 
-include::modules/virt-quick-creating-vm.adoc[leveloffset=+1]
+Templates without a boot source are labeled *Boot source required*. You can use these templates if you complete the steps for xref:../../virt/vm_templates/virt-creating-vm-template.adoc#virt-adding-a-boot-source-web_virt-creating-vm-template[adding a boot source to the virtual machine].
 
-include::modules/virt-creating-vm-custom-template.adoc[leveloffset=+1]
+include::modules/virt-quick-creating-vm.adoc[leveloffset=+2]
 
 Refer to the virtual machine fields section when creating a VM from the web console.
 
 :virtualmachine:
-include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
-
-include::modules/virt-storage-wizard-fields-web.adoc[leveloffset=+2]
-
-include::modules/virt-cloud-init-fields-web.adoc[leveloffset=+2]
-
-To configure storage class defaults, use storage profiles. For more information, see xref:../../virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc#virt-customizing-storage-profile_virt-creating-data-volumes[Customizing the storage profile].
+include::modules/virt-storage-wizard-fields-web.adoc[leveloffset=+3]
+:virtualmachine!:
```

Comment:
`:blah!:` is supported but `:!blah:` is [preferred](https://docs.asciidoctor.org/asciidoc/latest/attributes/unset-attributes/#unset-a-document-attribute-in-the-header) (and also far more common in the repo)

```suggestion
:!virtualmachine:
```